action-publish-edge/action.yaml

```diff
@@ -72,6 +72,7 @@ runs:
         printf "Running spread tests\n"
         wget -q https://storage.googleapis.com/snapd-spread-tests/spread/spread-plus-amd64.tar.gz \
             -O - | tar xz
+        mkdir -p ~/bin || true
```

Comment:
`|| true` is not needed afaict (-p will succeed if the dir already exists)

Reply:
Yes, you're right. Thanks, fixed!
Also, I moved the `mkdir` call up above the `wget` to keep it in-line with the existing `release` action.